pkgs/by-name/me/menulibre/package.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, gnome-menus
+, gtk3
+, intltool
+, gobject-introspection
+, wrapGAppsHook
+, testers
+, menulibre
+}:
+
+python3Packages.buildPythonPackage rec {
+  name = "menulibre";
+  version = "2.2.3";
+
+  src = fetchFromGitHub {
+    owner = "bluesabre";
+    repo = "menulibre";
+    rev = "menulibre-${version}";
+    hash = "sha256-E0ukq3q4YaakOI2mDs3dh0ncZX/dqspCA+97r3JwWyA=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    pygobject3
+    gnome-menus
+    psutil
+    distutils-extra
+  ];
+
+  nativeBuildInputs = [
+    gtk3
+    intltool
+    gobject-introspection
+    wrapGAppsHook
+  ];
+
+  postPatch = ''
+    sed -i "/__menulibre_data_directory__/s@(data_dir)@'$out/share/menulibre'@" setup.py
```

Comment:
Does it work?

Reply:
Yes, do you think it needs a comment explaining what it does? or maybe that this needs to use `patches`? I did not want to use `patches` because it makes it more tedious to update.

Reply:
@lelgenio optional: how do you feel about replacing this with `substituteInPlace` very common pattern in nixpkgs